model/src/main/kotlin/org/ostelco/prime/model/Entities.kt

```diff
@@ -332,6 +337,9 @@ data class RefundRecord(
         val reason: String, // possible values are duplicate, fraudulent, and requested_by_customer
         val timestamp: Long) : HasId
 
+/* TODO! (kmm) To support prorating with subscriptions, the
```

Comment:
what is "prorating"?

Reply:
Standard term for ensuring correct charges when changing/terminating subscriptions etc. More info at [https://stripe.com/docs/billing/subscriptions/prorations](https://stripe.com/docs/billing/subscriptions/prorations) for how to use it with Stripe. Also described many other places.